hax/hax/server.py

```diff
@@ -89,6 +89,21 @@ def hctl_stat(request):
     return json_response(text=result)
 
 
+def hctl_fetch_fids(request):
+    """This function calls the hare-fetch-fids script from the hax-server in
+    order to provide details about services configured by Hare e.g. Hax,
+    ios, confd, rgw etc
+    """
+    exec = Executor()
+    env = get_python_env()
+    result = exec.run(Program(["/opt/seagate/cortx/hare/libexec/"
```

Comment:
It would be good to have call a function instead of executing a script, let's create a improvisation ticket for the same.
This will also help us to write unit tests by mocking consul KVs.

Reply:
Created ticket https://jts.seagate.com/browse/CORTX-29396